packages/dds/map/src/directory.ts

```diff
@@ -336,6 +338,104 @@ export class DirectoryFactory implements IChannelFactory {
 	}
 }
 
+/**
+ *The comparator essentially performs the following procedure to determine the order of subdirectory creation:
+ * 1. If subdirectory A has a non-negative 'seq' and subdirectory B has a negative 'seq', subdirectory A is always placed first due to
+ * the policy that acknowledged subdirectories precede locally created ones that have not been committed yet.
+ *
+ * 2. If both subdirectories A and B have a non-negative 'seq', but subdirectory A has a lower 'clientSeq', subdirectory A will be
```

Comment:
The description for 2. is not quite right. If A and B have different non-negative 'seq', then they are compared by seq. If they have equal non-neg 'seq', then they are compared by 'clientSeq'.

Reply:
Similarly for 3.

Reply:
Updated